lib/broadway_cloud_pub_sub/pull_client.ex

```diff
@@ -173,6 +173,8 @@ defmodule BroadwayCloudPubSub.PullClient do
     %{message | "data" => Base.decode64!(encoded_data)}
   end
 
+  defp decode_message(message), do: message
```

Comment:
Shall we remove the first clause and check it is a binary in the second?

Reply:
Yep, that works :)

Reply:
@josevalim I added the is_binary guard to the first clause. Shall we check the "payloadFormat" attribute here, too, or do you think we're good as-is?

Reply:
I would say it would be semantically better but it is not a deal breaker. Your call!

Reply:
Looking back at #14 I think we need to keep the nil data match _and_ add this new clause, wdyt?

Reply:
Looks good to me!